extensions/notifiers/ifttt.js

```diff
@@ -0,0 +1,28 @@
+var request = require('request')
+
+module.exports = function container (get, set, clear) {
+  var ifttt = {
+    pushMessage: function(config, title, message) {
+      var postData = {'value1': title , 'value2': message }
+
+      function callback(error, response, body) {
+        if (!error) {
```

Comment:
I saw you already removed logs when no errors are reported in other notifiers, so I would do the same here, or at least, put it under c.debug.